hedera-node/hedera-app/src/main/java/module-info.java

```diff
@@ -52,6 +52,7 @@
     requires static com.github.spotbugs.annotations;
     requires static com.google.auto.service;
     requires static java.compiler;
+    requires static org.jetbrains.annotations;
```

Comment:
`edu.umd...` instead of `org.jetbrains...`

Reply:
Fixed.